apps/directory/src/components/report-components/ReportDetailsList.vue

```diff
@@ -21,7 +21,7 @@
         <td v-else-if="value.type === 'url'">
           <a :href="value.value" target="_blank" rel="noopener noreferrer">
             <i class="fa fa-fw fa-globe" aria-hidden="true"></i>
-            <span class="mg-icon-text">Website</span>
+            <span class="mg-icon-text">{{ value.label ? value.label : 'Website' }}</span>
```

Comment:
Instead of ternary you could use the js shorthand "value.label || 'Website'". Don't need to change it, but for future code ;)